xml/System.Threading/ManualResetEventSlim.xml

```diff
@@ -48,15 +48,15 @@
    
```

Comment:
Unrelated to your PR: The type description, "Provides a slimmed down version of ManualResetEvent", isn't helpful. Could you expand it by adding "to represent a thread synchronization event that, when signaled, must be reset manually."?